packages/swing-store/src/bundleStore.js

```diff
@@ -210,6 +209,16 @@ INSERT INTO bundles (bundleID, bundle) VALUES (?, NULL)
     }
   }
 
+  // take an export-data record (id/hash but not bundle contents) and
+  // insert something in the DB
+  function importBundleRecord(key, value) {
+    const [tag, ...pieces] = key.split('.');
+    assert.equal(tag, 'bundle');
+    const bundleID = pieces[0];
+    assert.equal(bundleID, value);
```

Comment:
I would prefer having an assertion about the length of `pieces`.
```suggestion
    assert.equal(pieces.length, 0);
    const bundleID = pieces[0];
    assert.equal(bundleID, value);
```

Reply:
except it needs to be `assert.equal(pieces.length, 1)`